docs/using-easybuild.md

```diff
@@ -528,6 +528,15 @@ For example, when looking for an easyconfig for `OpenMPI` version `4.1.4` and ve
 * `o/OpenMPI/OpenMPI-4.1.4-GCC-11.3.0-test.eb`
 * `OpenMPI-4.1.4-GCC-11.3.0-test.eb`
 
```

Comment:
Note that it is not an indefinite-depth search, so if you have cloned the easybuilders/easybuild-easyconfigs git repo then you need 'easybuilders/easybuild-easyconfigs/easybuild/easyconfigs' on your robot-path.

Reply:
I just added https://github.com/easybuilders/easybuild-docs/pull/258/commits/c7567cdcc160bc4eca0617a194fa4db4f1022803
Is that comprehensible enough?